### DISCUSSION TITLE
am i allowed to contribute to solidjs?

### DISCUSSION THREAD

**Discussion by rickyraz** [solidjs/solid#2314] [Ideas]
I'm new in this field, only 3 years of experience in programming and of those 3 years only 1 year working professionally..
can i contribute in solid js in terms of creating new hooks like useDefferedValue / useDebouce like the ones in reactjs.. ?

thx in advanced

**Comment by atk** [comment]
There's the solid-primitives community project. We already have scheduling primitives, but you can contribute other primitives.

**Reply by rickyraz** [reply]
ohh thx for the info .. idk yet before

**Reply by atk** [reply]
If you want to contribute, join our discord and ask on the #solid-primitives channel what would be helpful.
